src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointAllele.java

```diff
@@ -51,20 +68,56 @@ private InversionType getInversionType(final boolean fiveToThree, final boolean
         if (o == null || getClass() != o.getClass()) return false;
         final BreakpointAllele that = (BreakpointAllele) o;
-        return inversionType == that.inversionType &&
-                Objects.equals(leftAlignedLeftBreakpoint, that.leftAlignedLeftBreakpoint) &&
-                Objects.equals(leftAlignedRightBreakpoint, that.leftAlignedRightBreakpoint) &&
-                Objects.equals(insertedSequence, that.insertedSequence) &&
-                Objects.equals(homology, that.homology);
+
+        if (leftAlignedLeftBreakpoint != null ? !leftAlignedLeftBreakpoint.equals(that.leftAlignedLeftBreakpoint) : that.leftAlignedLeftBreakpoint != null)
+            return false;
+        if (leftAlignedRightBreakpoint != null ? !leftAlignedRightBreakpoint.equals(that.leftAlignedRightBreakpoint) : that.leftAlignedRightBreakpoint != null)
+            return false;
+        if (insertedSequence != null ? !insertedSequence.equals(that.insertedSequence) : that.insertedSequence != null)
+            return false;
+        if (homology != null ? !homology.equals(that.homology) : that.homology != null) return false;
+        return inversionType == that.inversionType;
+
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(leftAlignedLeftBreakpoint, leftAlignedRightBreakpoint, insertedSequence, homology, inversionType);
+        int result = leftAlignedLeftBreakpoint != null ? leftAlignedLeftBreakpoint.hashCode() : 0;
+        result = 31 * result + (inversionType != null ? inversionType.ordinal() + 1 : 0);
+        result = 31 * result + (leftAlignedRightBreakpoint != null ? leftAlignedRightBreakpoint.hashCode() : 0);
+        result = 31 * result + (insertedSequence != null ? insertedSequence.hashCode() : 0);
+        result = 31 * result + (homology != null ? homology.hashCode() : 0);
+        return result;
+    }
+
+    public static final class Serializer extends com.esotericsoftware.kryo.Serializer<BreakpointAllele> {
+        @Override
+        public void write(final Kryo kryo, final Output output, final BreakpointAllele breakpointAllele ) {
+            breakpointAllele.serialize(kryo, output);
+        }
+
+        @Override
+        public BreakpointAllele read(final Kryo kryo, final Input input, final Class<BreakpointAllele> klass ) {
+            return new BreakpointAllele(kryo, input);
+        }
+
+    }
+
+    private void serialize(final Kryo kryo, final Output output) {
+        output.writeString(leftAlignedLeftBreakpoint.getContig());
+        output.writeInt(leftAlignedLeftBreakpoint.getStart());
+        output.writeInt(leftAlignedLeftBreakpoint.getEnd());
+        output.writeString(leftAlignedRightBreakpoint.getContig());
+        output.writeInt(leftAlignedRightBreakpoint.getStart());
+        output.writeInt(leftAlignedRightBreakpoint.getEnd());
+        output.writeString(insertedSequence);
+        output.writeString(homology);
+        output.writeString(inversionType.toString());
```

Comment:
And if you take my suggestion to serialize inversionType as an int, this line would become:
`        output.writeString(inversionType.ordinal());`